sdks/go/pkg/beam/runners/flink/flink.go

```diff
@@ -31,8 +30,5 @@ func init() {
 // Execute runs the given pipeline on Flink. Convenience wrapper over the
 // universal runner.
 func Execute(ctx context.Context, p *beam.Pipeline) error {
-	if *jobopts.InternalJavaRunner == "" {
```

Comment:
I wonder if we should just remove the current file, there's nothing related to Flink anymore about it.

Reply:
I suspect we'll add some Flink-specific options and tweaks later.